dist/404.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
```

Comment:
Correct, the dist folder is to be deleted before this merges.

On Wed, Mar 2, 2016 at 4:43 PM, spra85 <notifications@github.com> wrote:

> In dist/404.html
> <https://github.com/theonion/bulbs-cms/pull/107#discussion_r54805677>:
>
> > @@ -0,0 +1,157 @@
> > +<!DOCTYPE html>
>
> Likewise, I don't think this was meant to be in the PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theonion/bulbs-cms/pull/107/files#r54805677>.
>